2026/selection-process.md

```diff
@@ -0,0 +1,183 @@
+# Interop 2026 proposal selection process
+
+Interop 2026 is an effort to increase interoperability across browsers
+in key technical areas that are of high priority to web developers and
+end users. This effort is part of the
+[web-platform-tests](https://github.com/web-platform-tests/wpt) (WPT)
+project — an automated test suite for web standards — and run by a
+team of representatives from companies that make substantial
+contributions to browser rendering engines (including Apple, Bocoup,
+Google, Igalia, Microsoft, and Mozilla).
+
+Please see the [Interop 2025 Dashboard](https://wpt.fyi/interop-2025)
+and [Interop 2025
+README](https://github.com/web-platform-tests/interop/blob/main/2025/README.md)
+for the current iteration of the project.
+
+Each iteration of the project starts with an open call for proposals,
+followed by a selection and prioritization process by each
+participating organization. Final decision on the inclusion of a
+proposal is made by
+[consensus](https://github.com/web-platform-tests/interop/blob/main/charter.md#:~:text=The%20team%20makes%20decisions%20based%20on%20consensus.%20A%20decision%20has%20consensus%20if%20it%20has%20support%20from%20at%20least%20two%20participating%20organizations%20and%20no%20opposition). For
+Interop 2026, the timeline and the selection process is as described
+below.
+
+## Timeline
+
+This is the general timeline for the Interop 2026 proposal selection 
+process. The dates are centered on the Interop team meeting (Thursdays).
+Please note that the timelines are subject to change.
+
+*   Proposal submission window (~3 weeks): Sept 18th, 2025 to Oct 8th, 2025
+*   Proposal selection: Before December 18th 2025
+*   Scope of the Interop 2026 project published: the first half of February, 2026.
+
+## Proposal selection process
+
+The aim of the proposal selection process is to generate a shared
+understanding of the merits of each proposal, and converge on a
+decision about the most impactful areas in which we can improve
+interop on the web over 2026.
+
+As in Interop 2024 and 2025, details of proposal selection and ranking
+remain participant confidential throughout the process. Final decisions
+will be published as the Interop team.
+
+Confidentiality does not prevent participating organizations from
+sharing their own priorities or roadmaps outside of the Interop program.
```

Comment:
After we launched Interop 2025 I commented on some proposals to give additional feedback, for example in https://github.com/web-platform-tests/interop/issues/755#issuecomment-2674354422.

I wasn't sure if commenting about our plans for https://crbug.com/40768696 on the proposal itself would be OK, so I'd like to clarify that here. The simplest edit I'd like to suggest is to remove the "outside the Interop program" part:

```suggestion
sharing their own priorities or roadmaps.
```

In other words, it's OK to share such information both during the process and to follow up later to say that a bug has been fixed or a feature shipped, for example. This information is often discoverable from a bug or something linked from the proposal anyway.

Reply:
I understand the concern, and think it comes down to the expectation that positions in the proposal selection process aren't revealed with a wink and a nudge. In past years we've proposed that confidentiality shouldn't apply to our own Interop positions, and that's still a change I'd welcome, but that's not the intention here.

Wording is hard, but the best I've been able to come up with is something like this:

> Confidentiality does not prevent participating organizations from sharing their own priorities or roadmaps, as distinct from positions in the proposal selection process.

I'd be OK with or without the extra clause, to me it's clear in either case that trying to find loopholes for confidentiality will not be well received and would be a breach of trust.

Reply:
This was discussed in the meeting today. @jgraham preferred the additional clause, so as an edit it would be this:

```suggestion
sharing their own priorities or roadmaps, as distinct from positions in the
proposal selection process.
```

@stubbornella can you take a look?